src/fidesops/service/privacy_request/request_runner_service.py

```diff
@@ -150,8 +151,22 @@ def queue_privacy_request(
     return task.task_id
 
 
-@celery_app.task()
+class DatabaseTask(Task):  # pylint: disable=W0223
+    _session = None
+
+    @property
+    def session(self) -> ContextManager[Session]:
+        """Creates Session once per process"""
+        if self._session is None:
+            SessionLocal = get_db_session(config)
+            self._session = SessionLocal()
+
+        return self._session
```

Comment:
This allows us to share the Session across a Celery process. 

This also limits the number of times `get_db_session` is being called per process which was creating new Engines which were opening up new Connection Pools whose connections weren't being reused.